addons/crm/views/crm_team_views.xml

```diff
@@ -176,23 +176,28 @@
                         </div>
                     </xpath>
 
-                    <xpath expr="//div[contains(@class, 'o_kanban_primary_right')]" position="inside">
-                        <div class="mb4" t-if="record.use_opportunities.raw_value">
-                            <a name="%(crm.action_report_crm_opportunity_salesteam)d" type="action">
-                                Pipeline Analysis
+                    <xpath expr="//t[@name='first_options']" position="after">
+                        <div class="row" t-if="record.unassigned_leads_count.raw_value">
+                            <a name="%(crm_case_form_view_salesteams_lead)d" type="action" context="{'search_default_unassigned': True}">
+                                <field name="unassigned_leads_count"/>
+                                <t t-if="record.unassigned_leads_count.raw_value == 1">Unassigned Lead</t>
+                                <t t-else="">Unassigned Leads</t>
                             </a>
                         </div>
-                        <div class="mb4" t-if="record.use_opportunities.raw_value">
-                            <a name="%(crm.crm_activity_report_action_team)d" type="action">
-                                Activities Analysis
-                            </a>
+                        <div class="row" t-if="record.opportunities_count.raw_value">
+                            <div class="col-xs-8">
+                                <a name="%(crm_case_form_view_salesteams_opportunity)d" type="action"> <!-- context="{'search_default_probability': NOT or < 100}" -->
+                                    <field name="opportunities_count"/>
+                                    <t t-if="record.opportunities_count.raw_value == 1">Open Opportunity</t>
```

Comment:
Indeed, it's intended.

0 things
1 thing
2 things

I believe it's how English works